ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bfd.yml

```diff
@@ -8,3 +8,12 @@ router_bfd:
     interval: 300
     min_rx: 300
     multiplier: 3
+  sbfd:
+    local_interface:
+      name: Loopback0
+      protocol: ipv4 ipv6
```

Comment:
Not a big fan of this data model. I would prefer something like:

```yaml
...
      protocols:
        ipv4: true
        ipv6: true
```

Reply:
Updated data model instead of using string